### ISSUE TITLE
ServerStatus failOSCFunc shouldn't set clientID

### ISSUE THREAD

**Issue by crucialfelix** [supercollider/supercollider#2328]
The OSC message sent on failure to register with scsynth for notifications returns a symbol 'too many users', not a clientID.

bug is here: https://github.com/supercollider/supercollider/blob/master/SCClassLibrary/Common/Control/ServerStatus.sc#L58

Stack trace is from atom-supercollider which is rich text and cannot be copy and pasted here. See pretty picture below.

```
Server:newBusAllocators
Args
this    
▶ Server localhost  
Vars
numControl  Integer 16384
numAudio    Integer 112
controlBusOffset    Symbol 'too many users'
audioBusOffset  Symbol 'too many users'
offset  Symbol 'too many users'
n   Integer 1
Server:newAllocators
Args
this    
▶ Server localhost  
Server:clientID_
Args
this    
▶ Server localhost  
val Symbol 'too many users'
a Function defined in undefined:undefined
Args
msg Array [ /fail, /notify, too many users ]
OSCArgsMatcher:value
Args
this    
▶ an OSCArgsMatcher  
testMsg Array [ /fail, /notify, too many users ]
time    Float 2.620803172
addr    
▶ a NetAddr(127.0.0.1, 57110)  
recvPort    Integer 57121
OSCFuncAddrMessageMatcher:value
Args
this    
▶ an OSCFuncAddrMessageMatcher  
msg Array [ /fail, /notify, too many users ]
time    Float 2.620803172
testAddr    
▶ a NetAddr(127.0.0.1, 57110)  
recvPort    Integer 57121
OSCMessageDispatcher:value
Args
this    
▶ an OSCMessageDispatcher  
msg Array [ /fail, /notify, too many users ]
time    Float 2.620803172
addr    
▶ a NetAddr(127.0.0.1, 57110)  
recvPort    Integer 57121
Main:recvOSCmessage
Args
this    a Main
time    Float 2.620803172
replyAddr   
▶ a NetAddr(127.0.0.1, 57110)  
recvPort    Integer 57121
msg Array [ /fail, /notify, too many users ]
```

![screen shot 2016-09-07 at 11 19 08](https://cloud.githubusercontent.com/assets/218738/18306671/2506a0e4-74ed-11e6-9c3d-73679302293a.png)

**Comment by vivid-synth** [comment]
Is this the same as https://github.com/supercollider/supercollider/issues/1914 ?

**Comment by crucialfelix** [comment]
Not the same as, but it is caused by that.  If it returns a symbol/string, then failOSCFunc is wrong.

Regardless of what it returns, if it failed then the clientID is invalid anyway and it should halt everything or something.

**Comment by crucialfelix** [comment]
Fixed by #2405
